### PR TITLE
Remove info about Mesos scheduler

### PR DESCRIPTION
**Pull request by objmagic**
cc @kramasamy

### CODE REVIEW

File: website/content/docs/contributors/custom-scheduler.md [hunk 1]
```diff
@@ -47,11 +46,11 @@ interfaces:
...
-[`IScheduler`](/api/com/twitter/heron/spi/scheduler/IScheduler.html) | Defines the scheduler object used to construct topologies | [Mesos](/api/scheduler/com/twitter/heron/scheduler/mesos/MesosScheduler.html), [local](/api/scheduler/com/twitter/heron/scheduler/local/LocalScheduler.html)
-[`IUploader`](/api/com/twitter/heron/spi/scheduler/IUploader.html) | Uploads the topology to a shared location that must be accessible to the runtime environment of the topology | [Aurora](), [Mesos](), [local](/api/scheduler/com/twitter/heron/scheduler/local/LocalUploader.html)
+[`IConfigLoader`](/api/com/twitter/heron/spi/scheduler/IConfigLoader.html) | Parsing and loading of configuration for the scheduler | [Aurora](/api/scheduler/com/twitter/heron/scheduler/aurora/AuroraConfigLoader.html), [local](/api/scheduler/com/twitter/heron/scheduler/local/LocalConfigLoader.html)
```
Comment (maosongfu) [inline website/content/docs/contributors/custom-scheduler.md]: The doc is a little outdated: IConfigLoader and IRuntimeManager is no longer there. You don't need to update them in this PR, but can u add an issue so we can trace and correct Scheduler-related docs?

**Comment by kramasamy** [comment]
👍

File: website/content/docs/contributors/custom-scheduler.md (hunk 1 above)
Comment (objmagic) [inline website/content/docs/contributors/custom-scheduler.md]: done, see #809

**Comment by samek** [comment]
So mesos scheduler is not supported anymore ?

**Comment by objmagic** [comment]
hi @samek , Mesos scheduler is still under development. See #803

**Comment by samek** [comment]
@objmagic Ok I see. 

Thanks, for the answer.

**Pull request merged**